### DISCUSSION TITLE
How to set up Multi Level Dropdown for Grid Inline Fields

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#7059] [Help]
I am trying to develop Multi Level Dropdown for inline grid fields like this 
https://www.w3schools.com/bootstrap/tryit.asp?filename=trybs_ref_js_dropdown_multilevel_css&stacked=h

For example, in the product demo (https://demo.serenity.is/Northwind/Product), the Category field would have multi level dropdown for Category and Sub-Category.

Has anyone done this before with Serenity? I tried a few things but was unable to get it to work

**Comment by furkanevran** [comment]
Hi @ibrahimGit123, you can refer to [this example](https://demo.serenity.is/AdvancedSamples/DragDropGrouping).
